Libbna/CUSTOM-CMS
# #192: Added an alternate text field for logo image

Steps to test:-
* Login as admin and go to Config page
* See if the alternate test field is visible
* If yes, fill the form and submit it
* Delete the image file for the current logo and refresh the page

Expected output:-
* The logo should disappear and the alternate text which you entered in the form should be displayed.
![image](https://user-images.githubusercontent.com/87715018/150143803-5413e683-e5e4-4caa-b996-a4ccae0e6833.png)


**Note:-** 
Created a new column in config table 'alt_text', make sure to update the database. Attached the sql file too.
